docs/integrations/sources/s3.md

```diff
@@ -45,10 +47,63 @@ At this time, object-level permissions alone are not sufficient to successfully
 :::
 
 4. Give your policy a descriptive name, then click **Create policy**.
-5. In the IAM dashboard, click **Users**. Select an existing IAM user or create a new one by clicking **Add users**.
-6. If you are using an _existing_ IAM user, click the **Add permissions** dropdown menu and select **Add permissions**. If you are creating a _new_ user, you will be taken to the Permissions screen after selecting a name.
-7. Select **Attach policies directly**, then find and check the box for your new policy. Click **Next**, then **Add permissions**.
-8. After successfully creating your user, select the **Security credentials** tab and click **Create access key**. You will be prompted to select a use case and add optional tags to your access key. Click **Create access key** to generate the keys.
+
+#### Option 1: Using an IAM Role (Most secure)
+
+1. In the IAM dashboard, click **Roles**, then **Create role**. <!-- env:oss -->
+2. Choose the appropriate trust entity and attach the policy you created. 
+3. Set up a trust relationship for the role. For example for **AWS account** trusted entity use default AWS account on your instance (it will be used to assume role). To use **External ID** set it to environment variables as `export AWS_EXTERNAL_ID="{your-external-id}"`. Edit the trust relationship policy to reflect this:
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::{your-aws-account-id}:user/{your-username}"
```

Comment:
We this is generally supported in cloud, this should probably say: arn:aws:iam::094410056844:user/delegated_access_user for the cloud version only.